api/download.go

```diff
@@ -71,6 +71,7 @@ func parseRequest(req *http.Request) (context.Context, string) {
 }
 
 func handleUnsupportedMetadataStates(ctx context.Context, m fclient.FileMetaData, cfg *config.Config, filePath string, w http.ResponseWriter) bool {
+	// If decrypted, which it will be?
```

Comment:
Does this need to change? I assume line 74 below is checking the file metadata has a `DECRYPTED` state We will remove the `DECRYPTED` state as part of this work. Should this be changed to `if files.Published(&m)` and we should perform the redirect when the file metadata state is `PUBLISHED` now?

Reply:
Removed state and unused functions